guide/profile.md

```diff
@@ -0,0 +1,46 @@
+## User Handbook
+### Your Profile
+
+Flarum has a basic profile system, in your profile there are 3 different categories of your profile that you can view.
+
+ - Posts
+ - Discussions
+ - Mentions
+
+## Categories Explained
```

Comment:
Yes, I agree with this one, I've changed the label from category to section on all of them.